### ISSUE TITLE
Confirmation Success Url

### ISSUE THREAD

**Issue by jclif** [lynndylanhurley/redux-auth#78]
Howdy!

I'm experiencing some issues with the confirmation flow. When I try to sign in as a new user, I am redirected back to the signup page but my app is throwing an error that I don't recognize. Has anyone encountered this particular issue?

Should I be setting the confirm_success_url within settings when I call `configure`?

**Comment by jclif** [comment]
I believe this might be related to #77.

**Comment by jclif** [comment]
@lynndylanhurley From what I can tell, the line:
```
value.replace(/[^+#$&/:<-\[\]-}]/g, encodeURIComponent)
```
fails when name is authHeaders, because value is undefined.

**Comment by betoharres** [comment]
it looks like the same problem as mine. Did you have this problem when trying to reset the password as well?

**Comment by vijaychouhan-rails** [comment]
Please set the `config.action_mailer.default_url_options = { :host => 'https://localhost:3000' }` in Rails application at development.rb

**Comment by jclif** [comment]
@vijaychouhan-rails I believe the rails server has the correct email url; it links me to my front-end application correctly.

**Comment by jclif** [comment]
@betoharres Yes, this happens on password change as well. What has been your solution to this? We would like to use this library for auth, but we are stuck because of these issues.

**Comment by betoharres** [comment]
@jclif Yeah, I did a PR 10 days ago fixing these problems, you can check it out my solution [here](https://github.com/lynndylanhurley/redux-auth/pull/86), but it was not merged. I'm using this lib locally with my modifications.
But the thing is: if you clone this repo right now in the master branch and run it's tests, it will show all green even with all these errors. When I fixed, I broke 24 tests but I fixed the problem and everything else works the same. 
If you know anything about the tests used in this lib, please help us out, it will be very welcome ❤️

**Comment by jclif** [comment]
@betoharres Interestingly, when I run these tests locally, they appear to pass without any problems.

**Comment by betoharres** [comment]
@jclif They do, that's the problem! haha
